FinSearchUnified/Tests/PluginTest.php

```diff
@@ -158,11 +204,9 @@ public function testEmptyValueNotAllowedExceptionIsThrownInExport()
     private function runExportAndReturnCount()
     {
         try {
-            $shopKey = 'ABCDABCDABCDABCDABCDABCDABCDABCD';
-            /** @var ShopwareProcess $blController */
-            $blController = Shopware()->Container()->get('fin_search_unified.shopware_process');
-            $blController->setShopKey($shopKey);
-            $xmlDocument = $blController->getFindologicXml();
+            $shopwareProcess = Shopware()->Container()->get('fin_search_unified.shopware_process');
```

Comment:
I think the docblock makes sense here. This way the IDE knows the class of the instance. But good idea to rename this variable name though :D.